RHEL/6/transforms/shorthand2xccdf.xslt

```diff
@@ -96,6 +96,12 @@
               </xsl:otherwise>
             </xsl:choose>
           </xsl:when>
+          <xsl:when test="name() = 'stig'">
+            <xsl:attribute name="system">
+              <xsl:value-of select="$cceuri" />
+            </xsl:attribute>
+            <xsl:value-of select="." />
+          </xsl:when>
```

Comment:
No problem with me. I just committed your suggested change.

On Wed, Feb 25, 2015 at 10:14 AM, Shawn Wells <notifications@github.com>
wrote:

> In RHEL/6/transforms/shorthand2xccdf.xslt
> <https://github.com/OpenSCAP/scap-security-guide/pull/454#discussion_r25352431>
> :
>
> > @@ -96,6 +96,12 @@
> >                </xsl:otherwise>
> >              </xsl:choose>
> >            </xsl:when>
> > +          <xsl:when test="name() = 'stig'">
> > +            <xsl:attribute name="system">
> > +              <xsl:value-of select="$cceuri" />
> > +            </xsl:attribute>
> > +            <xsl:value-of select="." />
> > +          </xsl:when>
>
> In the guide this will be displayed as:
>
> Security identifiers
>
> * CCE-26812-8
> * RHEL-06-000133
>
> What do you think of prepending "DISA FSO" ? e.g.:
>
> <xsl:value-of select="concat('CCE-', .)" />
>
> to
> <xsl:value-of select="concat('DISA FSO ',.)" />
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSCAP/scap-security-guide/pull/454/files#r25352431>
> .
>